### PR TITLE
Statically build Binary

### PR DESCRIPTION
**Pull request by cloudnautique**
Also, added Dockerfile.dist to have small docker image with the
socket proxy available.

### CODE REVIEW

File: scripts/build-image [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```
Comment (cjellick) [inline scripts/build-image:1]: So, where will this script be called?

File: Dockerfile.dist [hunk 2]
```diff
@@ -0,0 +1,4 @@
+FROM scratch
```
Comment (cjellick) [inline Dockerfile.dist:1]: Probably my ignorance here, but: do we have to have two separate dockerfiles?

File: scripts/build-image (hunk 1 above)
Comment (cloudnautique) [inline scripts/build-image:1]: Manually, when you want to build a container after a release for now. Will need a release script at some point. It is not terribly useful at the moment.

File: Dockerfile.dist (hunk 2 above)
Comment (cloudnautique) [inline Dockerfile.dist:1]: Yeah, the primary Dockerfile is setup for build, and the new one is for distribution. The differences are dramatic.

 From what I have seen on Google, and our own stuff with sherdock this approach seems to be the way to go.

**Comment by cjellick** [comment]
Ok @cloudnautique sounds good.
LGTM

**Pull request merged**